hello-buck-cxx/.buckconfig

```diff
@@ -0,0 +1,7 @@
+
+[cxx]
+cxxppflags= -D BUCKSTRING
+
+[cxx#zuck]
```

Comment:
Don't use zuck please. Also rather than doing ifdefs and the like below, just do `-DBUCK_PLATFORM="default"` and `-DBUCK_PLATFORM="other_platform"`